test/yaml/apiserver_partition/tenant1.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Tenant
 metadata:
   name: tenant1
+spec: 
+  storageClusterId: "1"
```

Comment:
To reviewers: do not worry about the scary "stop" sign added . The yaml was tested to work.  The sign just indicates no newline in the end.